### PR TITLE
Feature/wrapup

### LINKED ISSUE #18: Error handling - discussion
- Do we want a component for errors? Something consistent for all errors throughout the app?

### LINKED ISSUE #66: Add styling to the day of week for weekly forcast

### LINKED ISSUE #79: Deploy + Gifs + ReadMe
Roundin' the corner gang!!!!!

### LINKED ISSUE #80: Clean up SCSS
Gotta clean it up a bit, some stuff can be more dev-empathetic / SRP

### PR DESCRIPTION
**Pull request by leighlars**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

This PR fulfills the tests for `Header` and `Weekly Forecast`, adds a `README` + gifs, tidies up SCSS code and SCSS UX, and polishes off all UX so that our hard work is rendered as beautifully and as diligently as we worked.
 
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
Look at `Location`, `Weekly Forecast`, `Header`  

## Has This Been Tested?
- [ ] No
- [X] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
Look at  `Weekly Forecast`, `Header`  tests

# Checklist:
- [X] My code has no unused/commented out code
- [X] My code has no console logs
- [X] I have performed a self-review of my own code
- [X] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -42,39 +30,53 @@
...
 
 Users can view the air quality and weather information for American cities either by the search bar, or by perusing the home page for popular destination cities or cities with positive air quality rankings. Users can view a forecast of the coming week, and click the external links to read in-depth health, safety, and travel articles. 
 
-Time and free API calls were a hinderance, and our app is simply a proof of concept. If we had more time or more access to API data, we would have implemented allergy, pollen, and wildfire API data, and more information about which activities the user can partake in response to the current conditions.
+Time and free API calls were a hinderance, and our app is simply a proof of concept. If we had more time or more access to API data, we would have implemented allergy, pollen, and wildfire API data, and more information about which activities the user can partake in response to the current conditions. We would have also made the page mobile responsive.
```
Comment (ErinUntermeyer) [inline .github/README.md:39]: Great add on!

File: .github/README.md [hunk 2]
```diff
@@ -42,39 +30,53 @@
...
 Once you have locally cloned down the repo, within that file, run 
 `npm i`
 Once all dependencies are installed, run
 `npm start`
 
-
 ## Usage
 
-Use this space to show useful examples of how a project can be used. Additional screenshots, code examples and demos work well in this space. You may also link to more resources.
+![gif of home](./src/assets/homeload.gif)</br>
```
Comment (ErinUntermeyer) [inline .github/README.md:60]: Thanks for getting these done Leigh!

File: .github/README.md [hunk 3]
```diff
@@ -1,14 +1,4 @@
...
-[![Stargazers][stars-shield]][stars-url]
-[![Issues][issues-shield]][issues-url]
-[![LinkedIn][linkedin-shield]][linkedin-url]
-
-<br />
-<p align="center">
-  <a href="https://github.com/leighlars/WeatherOrNot">
-    <img src="images/logo.png" alt="Logo" width="80" height="80">
-  </a>
+
```
Comment (JoshSevy) [inline .github/README.md:1]: Dont like having badges on the site haha. Looks great

File: src/WeeklyForecast/WeeklyForecast.test.tsx [hunk 4]
```diff
@@ -1,147 +1,118 @@
...
 })
-xit("should render a section of the weekly forecast", () => {
 
- const { getByRole, getByText, getByAltText, getAllByText } = render(
-  <WeeklyForecast 
-    weeklyWeather={weeksWeather}
-  />
- );
- const weeklyHeader = getByRole("heading", { name: "WEEKLY FORECAST" });
+it("should render a section of the weekly forecast with multiple days", () => {
```
Comment (ErinUntermeyer) [inline src/WeeklyForecast/WeeklyForecast.test.tsx:64]: Great coverage on this test!

File: .github/README.md [hunk 5]
```diff
@@ -42,39 +30,53 @@
...
+![gif of home](./src/assets/homeload.gif)</br>
+*Home Page*
+
+![gif of location page](./src/assets/locationPage.gif)</br>
+*Location Page*
+
+![gif of about page](./src/assets/about.gif)</br>
+*About Page*
+
+![gif of search functionality](./src/assets/search.gif)</br>
```
Comment (JoshSevy) [inline .github/README.md:69]: Cool, I really like this. I will probably be stealing it for future projects.

File: src/Header/Header.test.tsx [hunk 6]
```diff
@@ -1,22 +1,55 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { render, screen, fireEvent, getByAltText } from "@testing-library/react";
```
Comment (JoshSevy) [inline src/Header/Header.test.tsx:2]: Just a heads up you dont have to import in getByAllText it lives on screen

**Comment by ErinUntermeyer** [comment]
Great work Leigh! The FINAL STRETCH!

**Review by ErinUntermeyer — APPROVED** [review]

File: src/Header/Header.test.tsx (hunk 6 above)
Comment (leighlars) [reply src/Header/Header.test.tsx:2]: what!!! maybe I was thinking of another time when Node yelled at me

File: .github/README.md (hunk 5 above)
Comment (leighlars) [reply .github/README.md:69]: omg I use it all the time

**Review by JoshSevy — APPROVED** [review]
This looks fantastic @leighlars Thank you for all the hard work. Could of asked for better partners.

**Pull request merged**